examples_utils/src/camera.rs

```diff
@@ -1,14 +1,17 @@
 use bevy::input::mouse::MouseMotion;
 use bevy::prelude::*;
 
+/// Simple flying camera plugin.
+/// In order to function, [`CameraController`] should be attached to the camera.
 pub struct CameraControllerPlugin;
 
 impl Plugin for CameraControllerPlugin {
     fn build(&self, app: &mut App) {
-        app.add_system(camera_controller);
+        app.add_system(camera_controller).add_system(print_controls);
```

Comment:
I don't think this should be a system. It should probably be a startup system

Reply:
Well, my idea was that controls can be changed in `CameraController` when it is attached to the camera. So we need a runtime to print current controls. On the other hand, default controls are fine, and I don't think they will be changed in examples.

Reply:
Yeah, but adding it in a system like that means it will print it on every frame. Which is just going to spam the console

Reply:
In that case I'm fine with it